change/@fluentui-react-charting-cd6618b3-30a0-474b-b2e9-9f008ff45c69.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1857 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26861&commitId=3aa2e0581e63a75e6743e9a1c545defe49410faa&env=prod)

✅ There was 1857 screenshots added, 0 screenshots removed, 0 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1857 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.disabled - Dark Mode.normal.chromium.png|0| Added|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|0| Added|
|Accordion Converged.disabled.normal.chromium.png|0| Added|
|Accordion Converged.expandIcon   Icon-   - Dark Mode.normal.chromium.png|0| Added|
|Accordion Converged.expandIcon   Icon-   - High Contrast.normal.chromium.png|0| Added|
|Accordion Converged.expandIcon   Icon-   - RTL.normal.chromium.png|0| Added|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|0| Added|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|0| Added|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|0| Added|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|0| Added|
|Accordion Converged.icon   Icon-  .normal.chromium.png|0| Added|
|Accordion Converged.size - RTL.normal.chromium.png|0| Added|
|Accordion Converged.size.normal.chromium.png|0| Added|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|0| Added|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|0| Added|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|0| Added|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|0| Added|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|0| Added|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|0| Added|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|0| Added|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|0| Added|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|0| Added|
|Accordion Converged.visibility+focus.focus closed.chromium.png|0| Added|
|Accordion Converged.visibility+focus.focus opened.chromium.png|0| Added|
|Accordion Converged.visibility+focus.normal.chromium.png|0| Added|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Added|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Added|
|Avatar Converged.basic.normal.chromium.png|0| Added|
|Avatar Converged.color - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.color - High Contrast.normal.chromium.png|0| Added|
|Avatar Converged.color+active - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.color+active - High Contrast.normal.chromium.png|0| Added|
|Avatar Converged.color+active.normal.chromium.png|0| Added|
|Avatar Converged.color.normal.chromium.png|0| Added|
|Avatar Converged.customSize+icon+active.normal.chromium.png|0| Added|
|Avatar Converged.customSize+image.normal.chromium.png|0| Added|
|Avatar Converged.customSize+name+badge.normal.chromium.png|0| Added|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|0| Added|
|Avatar Converged.image-bad-url.normal.chromium.png|0| Added|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|0| Added|
|Avatar Converged.size+image+badge.normal.chromium.png|0| Added|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Added|
|Avatar Converged.size+name.normal.chromium.png|0| Added|
|AvatarGroup Converged.basic - Dark Mode.default.chromium.png|0| Added|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|0| Added|
|AvatarGroup Converged.basic.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie - Dark Mode.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie - RTL.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-1 - Dark Mode.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-1 - RTL.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-1.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-2 - Dark Mode.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-2 - RTL.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie-2.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutPie.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|0| Added|
|AvatarGroup Converged.layoutStack.default.chromium.png|0| Added|
|AvatarGroup Converged.overflowContent - Dark Mode.chromium.png|0| Added|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|0| Added|
|AvatarGroup Converged.overflowContent - High Contrast.chromium.png|0| Added|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|0| Added|
|AvatarGroup Converged.overflowContent.chromium.png|0| Added|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|0| Added|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|0| Added|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|0| Added|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|0| Added|
|Badge Converged.filled - Dark Mode.chromium.png|0| Added|
|Badge Converged.filled - High Contrast.chromium.png|0| Added|
|Badge Converged.filled.chromium.png|0| Added|
|Badge Converged.ghost - Dark Mode.chromium.png|0| Added|
|Badge Converged.ghost - High Contrast.chromium.png|0| Added|
|Badge Converged.ghost.chromium.png|0| Added|
|Badge Converged.outline - Dark Mode.chromium.png|0| Added|
|Badge Converged.outline - High Contrast.chromium.png|0| Added|
|Badge Converged.outline.chromium.png|0| Added|
|Badge Converged.size- extra-large - RTL.chromium.png|0| Added|
|Badge Converged.size- extra-large.chromium.png|0| Added|
|Badge Converged.size- extra-small - RTL.chromium.png|0| Added|
|Badge Converged.size- extra-small.chromium.png|0| Added|
|Badge Converged.size- large - RTL.chromium.png|0| Added|
|Badge Converged.size- large.chromium.png|0| Added|
|Badge Converged.size- medium - RTL.chromium.png|0| Added|
|Badge Converged.size- medium.chromium.png|0| Added|
|Badge Converged.size- small - RTL.chromium.png|0| Added|
|Badge Converged.size- small.chromium.png|0| Added|
|Badge Converged.size- tiny - RTL.chromium.png|0| Added|
|Badge Converged.size- tiny.chromium.png|0| Added|
|Badge Converged.tint - Dark Mode.chromium.png|0| Added|
|Badge Converged.tint - High Contrast.chromium.png|0| Added|
|Badge Converged.tint.chromium.png|0| Added|
|Button Converged.As an anchor.default.chromium.png|0| Added|
|Button Converged.As an anchor.hover.chromium.png|0| Added|
|Button Converged.As an anchor.pressed.chromium.png|0| Added|
|Button Converged.Circular and icon only - RTL.default.chromium.png|0| Added|
|Button Converged.Circular and icon only - RTL.hover.chromium.png|0| Added|
|Button Converged.Circular and icon only - RTL.pressed.chromium.png|0| Added|
|Button Converged.Circular and icon only.default.chromium.png|0| Added|
|Button Converged.Circular and icon only.hover.chromium.png|0| Added|
|Button Converged.Circular and icon only.pressed.chromium.png|0| Added|
|Button Converged.Circular.default.chromium.png|0| Added|
|Button Converged.Circular.hover.chromium.png|0| Added|
|Button Converged.Circular.pressed.chromium.png|0| Added|
|Button Converged.Default - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Default - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Default - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Default - High Contrast.default.chromium.png|0| Added|
|Button Converged.Default - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Default - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Default - RTL.default.chromium.png|0| Added|
|Button Converged.Default - RTL.hover.chromium.png|0| Added|
|Button Converged.Default - RTL.pressed.chromium.png|0| Added|
|Button Converged.Default.default.chromium.png|0| Added|
|Button Converged.Default.hover.chromium.png|0| Added|
|Button Converged.Default.pressed.chromium.png|0| Added|
|Button Converged.Disabled - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Disabled - High Contrast.default.chromium.png|0| Added|
|Button Converged.Disabled - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Disabled - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Disabled.default.chromium.png|0| Added|
|Button Converged.Disabled.hover.chromium.png|0| Added|
|Button Converged.Disabled.pressed.chromium.png|0| Added|
|Button Converged.Icon only.default.chromium.png|0| Added|
|Button Converged.Icon only.hover.chromium.png|0| Added|
|Button Converged.Icon only.pressed.chromium.png|0| Added|
|Button Converged.Outline Disabled - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Outline Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Outline Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Outline Disabled - High Contrast.default.chromium.png|0| Added|
|Button Converged.Outline Disabled - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Outline Disabled - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Outline Disabled.default.chromium.png|0| Added|
|Button Converged.Outline Disabled.hover.chromium.png|0| Added|
|Button Converged.Outline Disabled.pressed.chromium.png|0| Added|
|Button Converged.Outline.default.chromium.png|0| Added|
|Button Converged.Outline.hover.chromium.png|0| Added|
|Button Converged.Outline.pressed.chromium.png|0| Added|
|Button Converged.Primary - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Primary - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Primary - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Primary - High Contrast.default.chromium.png|0| Added|
|Button Converged.Primary - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Primary - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Primary Disabled - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Primary Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Primary Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Primary Disabled - High Contrast.default.chromium.png|0| Added|
|Button Converged.Primary Disabled - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Primary Disabled - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Primary Disabled.default.chromium.png|0| Added|
|Button Converged.Primary Disabled.hover.chromium.png|0| Added|
|Button Converged.Primary Disabled.pressed.chromium.png|0| Added|
|Button Converged.Primary.default.chromium.png|0| Added|
|Button Converged.Primary.hover.chromium.png|0| Added|
|Button Converged.Primary.pressed.chromium.png|0| Added|
|Button Converged.Size large - with long text wrapping.chromium.png|0| Added|
|Button Converged.Size large.default.chromium.png|0| Added|
|Button Converged.Size large.hover.chromium.png|0| Added|
|Button Converged.Size large.pressed.chromium.png|0| Added|
|Button Converged.Size medium - with long text wrapping.chromium.png|0| Added|
|Button Converged.Size small - with long text wrapping.chromium.png|0| Added|
|Button Converged.Size small.default.chromium.png|0| Added|
|Button Converged.Size small.hover.chromium.png|0| Added|
|Button Converged.Size small.pressed.chromium.png|0| Added|
|Button Converged.Subtle - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Subtle - High Contrast.default.chromium.png|0| Added|
|Button Converged.Subtle - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Subtle Disabled - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Subtle Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Subtle Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Subtle Disabled - High Contrast.default.chromium.png|0| Added|
|Button Converged.Subtle Disabled - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Subtle Disabled - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Subtle Disabled.default.chromium.png|0| Added|
|Button Converged.Subtle Disabled.hover.chromium.png|0| Added|
|Button Converged.Subtle Disabled.pressed.chromium.png|0| Added|
|Button Converged.Subtle.default.chromium.png|0| Added|
|Button Converged.Subtle.hover.chromium.png|0| Added|
|Button Converged.Subtle.pressed.chromium.png|0| Added|
|Button Converged.Transparent - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Transparent - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Transparent - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Transparent - High Contrast.default.chromium.png|0| Added|
|Button Converged.Transparent - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Transparent - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Transparent Disabled - Dark Mode.default.chromium.png|0| Added|
|Button Converged.Transparent Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button Converged.Transparent Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button Converged.Transparent Disabled - High Contrast.default.chromium.png|0| Added|
|Button Converged.Transparent Disabled - High Contrast.hover.chromium.png|0| Added|
|Button Converged.Transparent Disabled - High Contrast.pressed.chromium.png|0| Added|
|Button Converged.Transparent Disabled.default.chromium.png|0| Added|
|Button Converged.Transparent Disabled.hover.chromium.png|0| Added|
|Button Converged.Transparent Disabled.pressed.chromium.png|0| Added|
|Button Converged.Transparent.default.chromium.png|0| Added|
|Button Converged.Transparent.hover.chromium.png|0| Added|
|Button Converged.Transparent.pressed.chromium.png|0| Added|
|Button Converged.With icon after content - RTL.default.chromium.png|0| Added|
|Button Converged.With icon after content - RTL.hover.chromium.png|0| Added|
|Button Converged.With icon after content - RTL.pressed.chromium.png|0| Added|
|Button Converged.With icon after content.default.chromium.png|0| Added|
|Button Converged.With icon after content.hover.chromium.png|0| Added|
|Button Converged.With icon after content.pressed.chromium.png|0| Added|
|Button Converged.With icon before content - RTL.default.chromium.png|0| Added|
|Button Converged.With icon before content - RTL.hover.chromium.png|0| Added|
|Button Converged.With icon before content - RTL.pressed.chromium.png|0| Added|
|Button Converged.With icon before content.default.chromium.png|0| Added|
|Button Converged.With icon before content.hover.chromium.png|0| Added|
|Button Converged.With icon before content.pressed.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|0| Added|
|Card Converged.CardHeader.normal.chromium.png|0| Added|
|Card Converged.appearance - Dark Mode.normal.chromium.png|0| Added|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Added|
|Card Converged.appearance - RTL.normal.chromium.png|0| Added|
|Card Converged.appearance.normal.chromium.png|0| Added|
|Card Converged.card templates.normal.chromium.png|0| Added|
|Card Converged.orientation.chromium.png|0| Added|
|Card Converged.size.normal.chromium.png|0| Added|
|Checkbox Converged.checked.active.chromium.png|0| Added|
|Checkbox Converged.checked.hover.chromium.png|0| Added|
|Checkbox Converged.checked.rest.chromium.png|0| Added|
|Checkbox Converged.circular+checked.default.chromium.png|0| Added|
|Checkbox Converged.circular+mixed.default.chromium.png|0| Added|
|Checkbox Converged.circular.default.chromium.png|0| Added|
|Checkbox Converged.disabled+checked.default.chromium.png|0| Added|
|Checkbox Converged.disabled+mixed.default.chromium.png|0| Added|
|Checkbox Converged.disabled.active.chromium.png|0| Added|
|Checkbox Converged.disabled.hover.chromium.png|0| Added|
|Checkbox Converged.disabled.rest.chromium.png|0| Added|
|Checkbox Converged.label-before - RTL.default.chromium.png|0| Added|
|Checkbox Converged.label-before.default.chromium.png|0| Added|
|Checkbox Converged.label-wrapping - RTL.default.chromium.png|0| Added|
|Checkbox Converged.label-wrapping.default.chromium.png|0| Added|
|Checkbox Converged.large - RTL.default.chromium.png|0| Added|
|Checkbox Converged.large+checked.default.chromium.png|0| Added|
|Checkbox Converged.large+circular+checked.default.chromium.png|0| Added|
|Checkbox Converged.large+circular+mixed.default.chromium.png|0| Added|
|Checkbox Converged.large+circular.default.chromium.png|0| Added|
|Checkbox Converged.large+label-wrapping - RTL.default.chromium.png|0| Added|
|Checkbox Converged.large+label-wrapping.default.chromium.png|0| Added|
|Checkbox Converged.large+mixed.default.chromium.png|0| Added|
|Checkbox Converged.large.default.chromium.png|0| Added|
|Checkbox Converged.mixed.active.chromium.png|0| Added|
|Checkbox Converged.mixed.hover.chromium.png|0| Added|
|Checkbox Converged.mixed.rest.chromium.png|0| Added|
|Checkbox Converged.no-label.default.chromium.png|0| Added|
|Checkbox Converged.required+label-before.default.chromium.png|0| Added|
|Checkbox Converged.required.default.chromium.png|0| Added|
|Checkbox Converged.unchecked - RTL.active.chromium.png|0| Added|
|Checkbox Converged.unchecked - RTL.hover.chromium.png|0| Added|
|Checkbox Converged.unchecked - RTL.rest.chromium.png|0| Added|
|Checkbox Converged.unchecked.active.chromium.png|0| Added|
|Checkbox Converged.unchecked.hover.chromium.png|0| Added|
|Checkbox Converged.unchecked.rest.chromium.png|0| Added|
|Combobox Converged.Appearance- filled-darker.default.chromium.png|0| Added|
|Combobox Converged.Appearance- filled-darker.focused.chromium.png|0| Added|
|Combobox Converged.Appearance- filled-darker.hover.chromium.png|0| Added|
|Combobox Converged.Appearance- filled-lighter.default.chromium.png|0| Added|
|Combobox Converged.Appearance- filled-lighter.focused.chromium.png|0| Added|
|Combobox Converged.Appearance- filled-lighter.hover.chromium.png|0| Added|
|Combobox Converged.Appearance- outline (default).default.chromium.png|0| Added|
|Combobox Converged.Appearance- outline (default).focused.chromium.png|0| Added|
|Combobox Converged.Appearance- outline (default).hover.chromium.png|0| Added|
|Combobox Converged.Appearance- underline.default.chromium.png|0| Added|
|Combobox Converged.Appearance- underline.focused.chromium.png|0| Added|
|Combobox Converged.Appearance- underline.hover.chromium.png|0| Added|
|Combobox Converged.Disabled option.active option.chromium.png|0| Added|
|Combobox Converged.Disabled option.default.chromium.png|0| Added|
|Combobox Converged.Disabled option.hover.chromium.png|0| Added|
|Combobox Converged.Disabled with value.default.chromium.png|0| Added|
|Combobox Converged.Disabled with value.focused.chromium.png|0| Added|
|Combobox Converged.Disabled with value.hover.chromium.png|0| Added|
|Combobox Converged.Disabled.default.chromium.png|0| Added|
|Combobox Converged.Disabled.focused.chromium.png|0| Added|
|Combobox Converged.Disabled.hover.chromium.png|0| Added|
|Combobox Converged.Invalid- filled-darker.default.chromium.png|0| Added|
|Combobox Converged.Invalid- filled-darker.focused.chromium.png|0| Added|
|Combobox Converged.Invalid- filled-darker.hover.chromium.png|0| Added|
|Combobox Converged.Invalid- filled-lighter.default.chromium.png|0| Added|
|Combobox Converged.Invalid- filled-lighter.focused.chromium.png|0| Added|
|Combobox Converged.Invalid- filled-lighter.hover.chromium.png|0| Added|
|Combobox Converged.Invalid- outline.default.chromium.png|0| Added|
|Combobox Converged.Invalid- outline.focused.chromium.png|0| Added|
|Combobox Converged.Invalid- outline.hover.chromium.png|0| Added|
|Combobox Converged.Invalid- underline.default.chromium.png|0| Added|
|Combobox Converged.Invalid- underline.focused.chromium.png|0| Added|
|Combobox Converged.Invalid- underline.hover.chromium.png|0| Added|
|Combobox Converged.Open with grouped options.active option.chromium.png|0| Added|
|Combobox Converged.Open with grouped options.default.chromium.png|0| Added|
|Combobox Converged.Open with grouped options.hover.chromium.png|0| Added|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|0| Added|
|Combobox Converged.Open with inlinePopup.default.chromium.png|0| Added|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|0| Added|
|Combobox Converged.Open.active option.chromium.png|0| Added|
|Combobox Converged.Open.default.chromium.png|0| Added|
|Combobox Converged.Open.hover.chromium.png|0| Added|
|Combobox Converged.Option with long content.active option.chromium.png|0| Added|
|Combobox Converged.Option with long content.default.chromium.png|0| Added|
|Combobox Converged.Option with long content.hover.chromium.png|0| Added|
|Combobox Converged.Size- large.default.chromium.png|0| Added|
|Combobox Converged.Size- large.focused.chromium.png|0| Added|
|Combobox Converged.Size- large.hover.chromium.png|0| Added|
|Combobox Converged.Size- small.default.chromium.png|0| Added|
|Combobox Converged.Size- small.focused.chromium.png|0| Added|
|Combobox Converged.Size- small.hover.chromium.png|0| Added|
|Combobox Converged.With multiselect selection.default.chromium.png|0| Added|
|Combobox Converged.With placeholder.default.chromium.png|0| Added|
|Combobox Converged.With placeholder.focused.chromium.png|0| Added|
|Combobox Converged.With placeholder.hover.chromium.png|0| Added|
|Combobox Converged.With selection.active option.chromium.png|0| Added|
|Combobox Converged.With selection.default.chromium.png|0| Added|
|Combobox Converged.With selection.hover.chromium.png|0| Added|
|Combobox Converged.With value.default.chromium.png|0| Added|
|Combobox Converged.With value.focused.chromium.png|0| Added|
|Combobox Converged.With value.hover.chromium.png|0| Added|
|CompoundButton Converged.Circular and icon only - RTL.default.chromium.png|0| Added|
|CompoundButton Converged.Circular and icon only - RTL.hover.chromium.png|0| Added|
|CompoundButton Converged.Circular and icon only - RTL.pressed.chromium.png|0| Added|
|CompoundButton Converged.Circular and icon only.default.chromium.png|0| Added|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|0| Added|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|0| Added|
|CompoundButton Converged.Circular.default.chromium.png|0| Added|
|CompoundButton Converged.Circular.hover.chromium.png|0| Added|
|CompoundButton Converged.Circular.pressed.chromium.png|0| Added|
|CompoundButton Converged.Default - RTL.default.chromium.png|0| Added|
|CompoundButton Converged.Default - RTL.pressed.chromium.png|0| Added|
|CompoundButton Converged.Default.default.chromium.png|0| Added|
|CompoundButton Converged.Default.pressed.chromium.png|0| Added|
|CompoundButton Converged.Disabled.default.chromium.png|0| Added|
|CompoundButton Converged.Disabled.hover.chromium.png|0| Added|
|CompoundButton Converged.Disabled.pressed.chromium.png|0| Added|
|CompoundButton Converged.Icon only.default.chromium.png|0| Added|
|CompoundButton Converged.Icon only.hover.chromium.png|0| Added|
|CompoundButton Converged.Icon only.pressed.chromium.png|0| Added|
|CompoundButton Converged.Outline Disabled.default.chromium.png|0| Added|
|CompoundButton Converged.Outline Disabled.hover.chromium.png|0| Added|
|CompoundButton Converged.Outline Disabled.pressed.chromium.png|0| Added|
|CompoundButton Converged.Outline.default.chromium.png|0| Added|
|CompoundButton Converged.Outline.hover.chromium.png|0| Added|
|CompoundButton Converged.Outline.pressed.chromium.png|0| Added|
|CompoundButton Converged.Primary Disabled.default.chromium.png|0| Added|
|CompoundButton Converged.Primary Disabled.hover.chromium.png|0| Added|
|CompoundButton Converged.Primary Disabled.pressed.chromium.png|0| Added|
|CompoundButton Converged.Primary.default.chromium.png|0| Added|
|CompoundButton Converged.Primary.hover.chromium.png|0| Added|
|CompoundButton Converged.Primary.pressed.chromium.png|0| Added|
|CompoundButton Converged.Size large - with long text wrapping.chromium.png|0| Added|
|CompoundButton Converged.Size large.default.chromium.png|0| Added|
|CompoundButton Converged.Size large.hover.chromium.png|0| Added|
|CompoundButton Converged.Size large.pressed.chromium.png|0| Added|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|0| Added|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|0| Added|
|CompoundButton Converged.Size small.default.chromium.png|0| Added|
|CompoundButton Converged.Size small.hover.chromium.png|0| Added|
|CompoundButton Converged.Size small.pressed.chromium.png|0| Added|
|CompoundButton Converged.Subtle Disabled.default.chromium.png|0| Added|
|CompoundButton Converged.Subtle Disabled.hover.chromium.png|0| Added|
|CompoundButton Converged.Subtle Disabled.pressed.chromium.png|0| Added|
|CompoundButton Converged.Subtle.default.chromium.png|0| Added|
|CompoundButton Converged.Subtle.hover.chromium.png|0| Added|
|CompoundButton Converged.Subtle.pressed.chromium.png|0| Added|
|CompoundButton Converged.Transparent Disabled.default.chromium.png|0| Added|
|CompoundButton Converged.Transparent Disabled.hover.chromium.png|0| Added|
|CompoundButton Converged.Transparent Disabled.pressed.chromium.png|0| Added|
|CompoundButton Converged.Transparent.default.chromium.png|0| Added|
|CompoundButton Converged.Transparent.hover.chromium.png|0| Added|
|CompoundButton Converged.Transparent.pressed.chromium.png|0| Added|
|CompoundButton Converged.With icon after content - RTL.default.chromium.png|0| Added|
|CompoundButton Converged.With icon after content - RTL.pressed.chromium.png|0| Added|
|CompoundButton Converged.With icon after content.default.chromium.png|0| Added|
|CompoundButton Converged.With icon after content.pressed.chromium.png|0| Added|
|CompoundButton Converged.With icon before content - RTL.default.chromium.png|0| Added|
|CompoundButton Converged.With icon before content - RTL.pressed.chromium.png|0| Added|
|CompoundButton Converged.With icon before content.default.chromium.png|0| Added|
|CompoundButton Converged.With icon before content.pressed.chromium.png|0| Added|
|CounterBadge Converged - colors.default - RTL.chromium.png|0| Added|
|CounterBadge Converged - colors.default.chromium.png|0| Added|
|Dialog.actions position start & position end - Dark Mode.chromium.png|0| Added|
|Dialog.actions position start & position end - High Contrast.chromium.png|0| Added|
|Dialog.actions position start & position end - RTL.chromium.png|0| Added|
|Dialog.actions position start & position end.chromium.png|0| Added|
|Dialog.actions position start - Dark Mode.chromium.png|0| Added|
|Dialog.actions position start - High Contrast.chromium.png|0| Added|
|Dialog.actions position start - RTL.chromium.png|0| Added|
|Dialog.actions position start.chromium.png|0| Added|
|Dialog.alert - Dark Mode.chromium.png|0| Added|
|Dialog.alert - High Contrast.chromium.png|0| Added|
|Dialog.alert - RTL.chromium.png|0| Added|
|Dialog.alert.chromium.png|0| Added|
|Dialog.default - Dark Mode.chromium.png|0| Added|
|Dialog.default - High Contrast.chromium.png|0| Added|
|Dialog.default - RTL.chromium.png|0| Added|
|Dialog.default.chromium.png|0| Added|
|Dialog.nested - Dark Mode.chromium.png|0| Added|
|Dialog.nested - High Contrast.chromium.png|0| Added|
|Dialog.nested - RTL.chromium.png|0| Added|
|Dialog.nested.chromium.png|0| Added|
|Dialog.no actions - Dark Mode.chromium.png|0| Added|
|Dialog.no actions - High Contrast.chromium.png|0| Added|
|Dialog.no actions - RTL.chromium.png|0| Added|
|Dialog.no actions.chromium.png|0| Added|
|Dialog.no title & no actions - Dark Mode.chromium.png|0| Added|
|Dialog.no title & no actions - High Contrast.chromium.png|0| Added|
|Dialog.no title & no actions - RTL.chromium.png|0| Added|
|Dialog.no title & no actions.chromium.png|0| Added|
|Dialog.no title - Dark Mode.chromium.png|0| Added|
|Dialog.no title - High Contrast.chromium.png|0| Added|
|Dialog.no title - RTL.chromium.png|0| Added|
|Dialog.no title.chromium.png|0| Added|
|Dialog.non-modal - Dark Mode.chromium.png|0| Added|
|Dialog.non-modal - High Contrast.chromium.png|0| Added|
|Dialog.non-modal - RTL.chromium.png|0| Added|
|Dialog.non-modal.chromium.png|0| Added|
|Dialog.scroll long content - Dark Mode.chromium.png|0| Added|
|Dialog.scroll long content - High Contrast.chromium.png|0| Added|
|Dialog.scroll long content - RTL.chromium.png|0| Added|
|Dialog.scroll long content.chromium.png|0| Added|
|Dialog.title custom action - Dark Mode.chromium.png|0| Added|
|Dialog.title custom action - High Contrast.chromium.png|0| Added|
|Dialog.title custom action - RTL.chromium.png|0| Added|
|Dialog.title custom action.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance brand - Dark Mode.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance brand - High Contrast.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance brand.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance strong - High Contrast.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance strong.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance subtle - High Contrast.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Appearance subtle.default.chromium.png|0| Added|
|Divider Converged - Horizontal.End Aligned - RTL.default.chromium.png|0| Added|
|Divider Converged - Horizontal.End Aligned.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Inset.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Start Aligned - RTL.default.chromium.png|0| Added|
|Divider Converged - Horizontal.Start Aligned.default.chromium.png|0| Added|
|Divider Converged - Horizontal.with content - Dark Mode.default.chromium.png|0| Added|
|Divider Converged - Horizontal.with content - High Contrast.default.chromium.png|0| Added|
|Divider Converged - Horizontal.with content - RTL.default.chromium.png|0| Added|
|Divider Converged - Horizontal.with content.default.chromium.png|0| Added|
|Divider Converged - Horizontal.without content - RTL.default.chromium.png|0| Added|
|Divider Converged - Horizontal.without content.default.chromium.png|0| Added|
|Divider Converged - Vertical.Center Aligned.default.chromium.png|0| Added|
|Divider Converged - Vertical.End Aligned.default.chromium.png|0| Added|
|Divider Converged - Vertical.Start Aligned.default.chromium.png|0| Added|
|Divider Converged - Vertical.inset.default.chromium.png|0| Added|
|Dropdown Converged.Appearance- filled-darker.default.chromium.png|0| Added|
|Dropdown Converged.Appearance- filled-lighter.default.chromium.png|0| Added|
|Dropdown Converged.Appearance- outline (default).default.chromium.png|0| Added|
|Dropdown Converged.Appearance- underline.default.chromium.png|0| Added|
|Dropdown Converged.Disabled option.active option.chromium.png|0| Added|
|Dropdown Converged.Disabled option.default.chromium.png|0| Added|
|Dropdown Converged.Disabled option.hover.chromium.png|0| Added|
|Dropdown Converged.Disabled with value.default.chromium.png|0| Added|
|Dropdown Converged.Disabled.default.chromium.png|0| Added|
|Dropdown Converged.Invalid- filled-darker.default.chromium.png|0| Added|
|Dropdown Converged.Invalid- filled-lighter.default.chromium.png|0| Added|
|Dropdown Converged.Invalid- outline.default.chromium.png|0| Added|
|Dropdown Converged.Invalid- underline.default.chromium.png|0| Added|
|Dropdown Converged.Open with grouped options.active option.chromium.png|0| Added|
|Dropdown Converged.Open with grouped options.default.chromium.png|0| Added|
|Dropdown Converged.Open with grouped options.hover.chromium.png|0| Added|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|0| Added|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|0| Added|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|0| Added|
|Dropdown Converged.Open.active option.chromium.png|0| Added|
|Dropdown Converged.Open.default.chromium.png|0| Added|
|Dropdown Converged.Open.hover.chromium.png|0| Added|
|Dropdown Converged.Option with long content.active option.chromium.png|0| Added|
|Dropdown Converged.Option with long content.default.chromium.png|0| Added|
|Dropdown Converged.Option with long content.hover.chromium.png|0| Added|
|Dropdown Converged.Size- large.default.chromium.png|0| Added|
|Dropdown Converged.Size- small.default.chromium.png|0| Added|
|Dropdown Converged.With multiselect selection.default.chromium.png|0| Added|
|Dropdown Converged.With multiselect value.default.chromium.png|0| Added|
|Dropdown Converged.With placeholder.default.chromium.png|0| Added|
|Dropdown Converged.With selection.active option.chromium.png|0| Added|
|Dropdown Converged.With selection.default.chromium.png|0| Added|
|Dropdown Converged.With selection.hover.chromium.png|0| Added|
|Dropdown Converged.With value.default.chromium.png|0| Added|
|Field.Checkbox-error.default.chromium.png|0| Added|
|Field.Combobox-error.default.chromium.png|0| Added|
|Field.Dropdown-error.default.chromium.png|0| Added|
|Field.ProgressBar-error.default.chromium.png|0| Added|
|Field.RadioGroup-error.default.chromium.png|0| Added|
|Field.Select-error.default.chromium.png|0| Added|
|Field.Slider-error.default.chromium.png|0| Added|
|Field.SpinButton-error.default.chromium.png|0| Added|
|Field.Switch-error.default.chromium.png|0| Added|
|Field.Textarea-error.default.chromium.png|0| Added|
|Field.base.default.chromium.png|0| Added|
|Field.hint.default.chromium.png|0| Added|
|Field.horizontal+longLabel.default.chromium.png|0| Added|
|Field.horizontal+noLabel.default.chromium.png|0| Added|
|Field.horizontal.default.chromium.png|0| Added|
|Field.required.default.chromium.png|0| Added|
|Field.size-large.default.chromium.png|0| Added|
|Field.size-small.default.chromium.png|0| Added|
|Field.validation-error.default.chromium.png|0| Added|
|Field.validation-none.default.chromium.png|0| Added|
|Field.validation-success.default.chromium.png|0| Added|
|Field.validation-warning.default.chromium.png|0| Added|
|Field.validationMessageIcon.default.chromium.png|0| Added|
|Image Converged.Default.normal.chromium.png|0| Added|
|Image Converged.Image Fluid.normal.chromium.png|0| Added|
|Image Converged.Image Layout Fit.normal.chromium.png|0| Added|
|Image Converged.Image Shadow.normal.chromium.png|0| Added|
|Image Converged.Image Shape.normal.chromium.png|0| Added|
|Image Converged.Image Variations Border.normal.chromium.png|0| Added|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Added|
|InfoButton.default - Dark Mode.active.chromium.png|0| Added|
|InfoButton.default - Dark Mode.focus.chromium.png|0| Added|
|InfoButton.default - Dark Mode.hover.chromium.png|0| Added|
|InfoButton.default - Dark Mode.rest.chromium.png|0| Added|
|InfoButton.default - High Contrast.active.chromium.png|0| Added|
|InfoButton.default - High Contrast.focus.chromium.png|0| Added|
|InfoButton.default - High Contrast.hover.chromium.png|0| Added|
|InfoButton.default - High Contrast.rest.chromium.png|0| Added|
|InfoButton.default.active.chromium.png|0| Added|
|InfoButton.default.focus.chromium.png|0| Added|
|InfoButton.default.hover.chromium.png|0| Added|
|InfoButton.default.rest.chromium.png|0| Added|
|InfoButton.size.rest.chromium.png|0| Added|
|Input Converged.Appearance- filled-darker.default.chromium.png|0| Added|
|Input Converged.Appearance- filled-darker.focused.chromium.png|0| Added|
|Input Converged.Appearance- filled-darker.hover.chromium.png|0| Added|
|Input Converged.Appearance- filled-lighter.default.chromium.png|0| Added|
|Input Converged.Appearance- filled-lighter.focused.chromium.png|0| Added|
|Input Converged.Appearance- filled-lighter.hover.chromium.png|0| Added|
|Input Converged.Appearance- outline (default).default.chromium.png|0| Added|
|Input Converged.Appearance- outline (default).focused.chromium.png|0| Added|
|Input Converged.Appearance- outline (default).hover.chromium.png|0| Added|
|Input Converged.Appearance- underline.default.chromium.png|0| Added|
|Input Converged.Appearance- underline.focused.chromium.png|0| Added|
|Input Converged.Appearance- underline.hover.chromium.png|0| Added|
|Input Converged.Disabled, appearance- filled-darker.default.chromium.png|0| Added|
|Input Converged.Disabled, appearance- filled-darker.focused.chromium.png|0| Added|
|Input Converged.Disabled, appearance- filled-darker.hover.chromium.png|0| Added|
|Input Converged.Disabled, appearance- filled-lighter.default.chromium.png|0| Added|
|Input Converged.Disabled, appearance- filled-lighter.focused.chromium.png|0| Added|
|Input Converged.Disabled, appearance- filled-lighter.hover.chromium.png|0| Added|
|Input Converged.Disabled, appearance- outline.default.chromium.png|0| Added|
|Input Converged.Disabled, appearance- outline.focused.chromium.png|0| Added|
|Input Converged.Disabled, appearance- outline.hover.chromium.png|0| Added|
|Input Converged.Disabled, appearance- underline.default.chromium.png|0| Added|
|Input Converged.Disabled, appearance- underline.focused.chromium.png|0| Added|
|Input Converged.Disabled, appearance- underline.hover.chromium.png|0| Added|
|Input Converged.Inline.default.chromium.png|0| Added|
|Input Converged.Invalid, appearance- filled-darker.default.chromium.png|0| Added|
|Input Converged.Invalid, appearance- filled-darker.focused.chromium.png|0| Added|
|Input Converged.Invalid, appearance- filled-darker.hover.chromium.png|0| Added|
|Input Converged.Invalid, appearance- filled-lighter.default.chromium.png|0| Added|
|Input Converged.Invalid, appearance- filled-lighter.focused.chromium.png|0| Added|
|Input Converged.Invalid, appearance- filled-lighter.hover.chromium.png|0| Added|
|Input Converged.Invalid, appearance- outline.default.chromium.png|0| Added|
|Input Converged.Invalid, appearance- outline.focused.chromium.png|0| Added|
|Input Converged.Invalid, appearance- outline.hover.chromium.png|0| Added|
|Input Converged.Invalid, appearance- underline.default.chromium.png|0| Added|
|Input Converged.Invalid, appearance- underline.focused.chromium.png|0| Added|
|Input Converged.Invalid, appearance- underline.hover.chromium.png|0| Added|
|Input Converged.Size- large.default.chromium.png|0| Added|
|Input Converged.Size- small.default.chromium.png|0| Added|
|Input Converged.With appearance override.default.chromium.png|0| Added|
|Input Converged.With value.default.chromium.png|0| Added|
|Input Converged.With value.focused.chromium.png|0| Added|
|Input Converged.With value.hover.chromium.png|0| Added|
|Input Converged.contentAfter - RTL.default.chromium.png|0| Added|
|Input Converged.contentAfter.default.chromium.png|0| Added|
|Input Converged.contentBefore - RTL.default.chromium.png|0| Added|
|Input Converged.contentBefore.default.chromium.png|0| Added|
|Label Converged.CustomRequired - RTL.default.chromium.png|0| Added|
|Label Converged.CustomRequired.default.chromium.png|0| Added|
|Label Converged.Disabled - Dark Mode.default.chromium.png|0| Added|
|Label Converged.Disabled - High Contrast.default.chromium.png|0| Added|
|Label Converged.Disabled.default.chromium.png|0| Added|
|Label Converged.Large.default.chromium.png|0| Added|
|Label Converged.Multiline - RTL.default.chromium.png|0| Added|
|Label Converged.Multiline.default.chromium.png|0| Added|
|Label Converged.Required - Dark Mode.default.chromium.png|0| Added|
|Label Converged.Required - High Contrast.default.chromium.png|0| Added|
|Label Converged.Required.default.chromium.png|0| Added|
|Label Converged.Root - Dark Mode.default.chromium.png|0| Added|
|Label Converged.Root - High Contrast.default.chromium.png|0| Added|
|Label Converged.Root - RTL.default.chromium.png|0| Added|
|Label Converged.Root.default.chromium.png|0| Added|
|Label Converged.Semibold.default.chromium.png|0| Added|
|Label Converged.Small.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline - RTL.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline - RTL.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline - RTL.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline - RTL.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline Disabled.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Inline.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - RTL.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - RTL.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - RTL.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone - RTL.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone Disabled.pressed.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone.default.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone.focused.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone.hover.chromium.png|0| Added|
|Link Converged - Rendered as anchor.Stand-alone.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline - RTL.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline - RTL.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline - RTL.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline - RTL.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled Focusable.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled Focusable.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled Focusable.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled Focusable.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline Disabled.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Inline.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - RTL.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - RTL.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - RTL.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - RTL.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone.pressed.chromium.png|0| Added|
|MakeStyles (pseudo).insertion is ordered.boxA, focus+hover.chromium.png|0| Added|
|MakeStyles (pseudo).insertion is ordered.boxA, focus.chromium.png|0| Added|
|MakeStyles (pseudo).insertion is ordered.boxB, focus+hover.chromium.png|0| Added|
|MakeStyles (pseudo).insertion is ordered.normal.chromium.png|0| Added|
|MakeStyles.RTL- two components in a single Provider.normal.chromium.png|0| Added|
|MakeStyles.RTL- two providers with different directions as children.normal.chromium.png|0| Added|
|MakeStyles.RTL- two providers with different directions as siblings.normal.chromium.png|0| Added|
|Menu Converged - basic.default - RTL.chromium.png|0| Added|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged - basic.default.chromium.png|0| Added|
|Menu Converged - basic.default.hover menuitem.chromium.png|0| Added|
|Menu Converged - groups.default - Dark Mode.chromium.png|0| Added|
|Menu Converged - groups.default - High Contrast.chromium.png|0| Added|
|Menu Converged - groups.default - RTL.chromium.png|0| Added|
|Menu Converged - groups.default.chromium.png|0| Added|
|Menu Converged - icon slotted content.default - RTL.chromium.png|0| Added|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged - icon slotted content.default.chromium.png|0| Added|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|0| Added|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|0| Added|
|Menu Converged - nested submenus.default - RTL.chromium.png|0| Added|
|Menu Converged - nested submenus.default.all open.chromium.png|0| Added|
|Menu Converged - nested submenus.default.chromium.png|0| Added|
|Menu Converged - secondary content.default - RTL.chromium.png|0| Added|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged - secondary content.default.chromium.png|0| Added|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|0| Added|
|Menu Converged - selection groups.default - RTL.chromium.png|0| Added|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|0| Added|
|Menu Converged - selection groups.default.chromium.png|0| Added|
|Menu Converged - selection groups.default.selected.chromium.png|0| Added|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|0| Added|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|0| Added|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|0| Added|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|0| Added|
|Menu Converged - selection.checkbox - RTL.chromium.png|0| Added|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|0| Added|
|Menu Converged - selection.checkbox.chromium.png|0| Added|
|Menu Converged - selection.checkbox.selected.chromium.png|0| Added|
|Menu Converged - split item.default - RTL.chromium.png|0| Added|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|0| Added|
|Menu Converged - split item.default.chromium.png|0| Added|
|Menu Converged - split item.default.submenu open.chromium.png|0| Added|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|0| Added|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|0| Added|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|0| Added|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|0| Added|
|Menu nested within a MenuTrigger.default.chromium.png|0| Added|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|0| Added|
|MenuButton Converged.Circular and icon only.default.chromium.png|0| Added|
|MenuButton Converged.Circular and icon only.hover.chromium.png|0| Added|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|0| Added|
|MenuButton Converged.Circular.default.chromium.png|0| Added|
|MenuButton Converged.Circular.hover.chromium.png|0| Added|
|MenuButton Converged.Circular.pressed.chromium.png|0| Added|
|MenuButton Converged.Default - RTL.default.chromium.png|0| Added|
|MenuButton Converged.Default - RTL.hover.chromium.png|0| Added|
|MenuButton Converged.Default - RTL.pressed.chromium.png|0| Added|
|MenuButton Converged.Default.default.chromium.png|0| Added|
|MenuButton Converged.Default.hover.chromium.png|0| Added|
|MenuButton Converged.Default.pressed.chromium.png|0| Added|
|MenuButton Converged.Disabled.default.chromium.png|0| Added|
|MenuButton Converged.Disabled.hover.chromium.png|0| Added|
|MenuButton Converged.Disabled.pressed.chromium.png|0| Added|
|MenuButton Converged.Icon only.default.chromium.png|0| Added|
|MenuButton Converged.Icon only.hover.chromium.png|0| Added|
|MenuButton Converged.Icon only.pressed.chromium.png|0| Added|
|MenuButton Converged.Outline Disabled.default.chromium.png|0| Added|
|MenuButton Converged.Outline Disabled.hover.chromium.png|0| Added|
|MenuButton Converged.Outline Disabled.pressed.chromium.png|0| Added|
|MenuButton Converged.Outline.default.chromium.png|0| Added|
|MenuButton Converged.Outline.hover.chromium.png|0| Added|
|MenuButton Converged.Outline.pressed.chromium.png|0| Added|
|MenuButton Converged.Primary Disabled.default.chromium.png|0| Added|
|MenuButton Converged.Primary Disabled.hover.chromium.png|0| Added|
|MenuButton Converged.Primary Disabled.pressed.chromium.png|0| Added|
|MenuButton Converged.Primary.default.chromium.png|0| Added|
|MenuButton Converged.Primary.hover.chromium.png|0| Added|
|MenuButton Converged.Primary.pressed.chromium.png|0| Added|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|0| Added|
|MenuButton Converged.Size large.default.chromium.png|0| Added|
|MenuButton Converged.Size large.hover.chromium.png|0| Added|
|MenuButton Converged.Size large.pressed.chromium.png|0| Added|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|0| Added|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|0| Added|
|MenuButton Converged.Size small.default.chromium.png|0| Added|
|MenuButton Converged.Size small.hover.chromium.png|0| Added|
|MenuButton Converged.Size small.pressed.chromium.png|0| Added|
|MenuButton Converged.Subtle Disabled.default.chromium.png|0| Added|
|MenuButton Converged.Subtle Disabled.hover.chromium.png|0| Added|
|MenuButton Converged.Subtle Disabled.pressed.chromium.png|0| Added|
|MenuButton Converged.Subtle.default.chromium.png|0| Added|
|MenuButton Converged.Subtle.hover.chromium.png|0| Added|
|MenuButton Converged.Subtle.pressed.chromium.png|0| Added|
|MenuButton Converged.Transparent Disabled.default.chromium.png|0| Added|
|MenuButton Converged.Transparent Disabled.hover.chromium.png|0| Added|
|MenuButton Converged.Transparent Disabled.pressed.chromium.png|0| Added|
|MenuButton Converged.Transparent.default.chromium.png|0| Added|
|MenuButton Converged.Transparent.hover.chromium.png|0| Added|
|MenuButton Converged.Transparent.pressed.chromium.png|0| Added|
|MenuButton Converged.With icon.default.chromium.png|0| Added|
|MenuButton Converged.With icon.hover.chromium.png|0| Added|
|MenuButton Converged.With icon.pressed.chromium.png|0| Added|
|Persona Converged.basic - Dark Mode.normal.chromium.png|0| Added|
|Persona Converged.basic - High Contrast.normal.chromium.png|0| Added|
|Persona Converged.basic - RTL.normal.chromium.png|0| Added|
|Persona Converged.basic.normal.chromium.png|0| Added|
|Persona Converged.size-avatar.normal.chromium.png|0| Added|
|Persona Converged.size-presence.normal.chromium.png|0| Added|
|Persona Converged.textAlignment.normal.chromium.png|0| Added|
|Persona Converged.textPosition.normal.chromium.png|0| Added|
|Persona Converged.textWrap.normal.chromium.png|0| Added|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|0| Added|
|Popover Converged.avoid scrolling.chromium.png|0| Added|
|Popover Converged.positioning - High Contrast.chromium.png|0| Added|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|0| Added|
|Popover Converged.positioning - RTL.chromium.png|0| Added|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|0| Added|
|Popover Converged.positioning.chromium.png|0| Added|
|Popover Converged.positioning.positioned popovers.chromium.png|0| Added|
|Portal.Apply Class Names.chromium.png|0| Added|
|Portal.Apply Class Names.should have green border.chromium.png|0| Added|
|Positioning.arrow - RTL.chromium.png|0| Added|
|Positioning.arrow.chromium.png|0| Added|
|Positioning.auto size.chromium.png|0| Added|
|Positioning.coverTarget - RTL.chromium.png|0| Added|
|Positioning.coverTarget.chromium.png|0| Added|
|Positioning.disable tether.chromium.png|0| Added|
|Positioning.explicit overflow padding - RTL.chromium.png|0| Added|
|Positioning.explicit overflow padding.chromium.png|0| Added|
|Positioning.horizontal flip - RTL.chromium.png|0| Added|
|Positioning.horizontal flip.chromium.png|0| Added|
|Positioning.horizontal overflow - RTL.chromium.png|0| Added|
|Positioning.horizontal overflow padding - RTL.chromium.png|0| Added|
|Positioning.horizontal overflow padding.chromium.png|0| Added|
|Positioning.horizontal overflow.chromium.png|0| Added|
|Positioning.imperative target.chromium.png|0| Added|
|Positioning.offset - RTL.chromium.png|0| Added|
|Positioning.offset function - RTL.chromium.png|0| Added|
|Positioning.offset function.chromium.png|0| Added|
|Positioning.offset.chromium.png|0| Added|
|Positioning.pinned.chromium.png|0| Added|
|Positioning.position and align props - RTL.chromium.png|0| Added|
|Positioning.position and align props.chromium.png|0| Added|
|Positioning.position fixed - RTL.chromium.png|0| Added|
|Positioning.position fixed.chromium.png|0| Added|
|Positioning.reset target.chromium.png|0| Added|
|Positioning.target property.chromium.png|0| Added|
|Positioning.vertical flip.chromium.png|0| Added|
|Positioning.vertical overflow padding.chromium.png|0| Added|
|Positioning.vertical overflow.chromium.png|0| Added|
|Positioning.virtual element.chromium.png|0| Added|
|Positioning.visibility modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|0| Added|
|Positioning.visibility modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|0| Added|
|Positioning.visibility modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|0| Added|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|0| Added|
|PresenceBadge Converged - OOF status.default.chromium.png|0| Added|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|0| Added|
|PresenceBadge Converged - sizes.default.chromium.png|0| Added|
|PresenceBadge Converged - status.default - RTL.chromium.png|0| Added|
|PresenceBadge Converged - status.default.chromium.png|0| Added|
|ProgressBar converged.Determinate - Dark Mode.chromium.png|0| Added|
|ProgressBar converged.Determinate - High Contrast.chromium.png|0| Added|
|ProgressBar converged.Determinate - RTL.chromium.png|0| Added|
|ProgressBar converged.Determinate with thickness large.chromium.png|0| Added|
|ProgressBar converged.Determinate.chromium.png|0| Added|
|ProgressBar converged.Error.chromium.png|0| Added|
|ProgressBar converged.Indeterminate - Dark Mode.chromium.png|0| Added|
|ProgressBar converged.Indeterminate - High Contrast.chromium.png|0| Added|
|ProgressBar converged.Indeterminate - RTL.chromium.png|0| Added|
|ProgressBar converged.Indeterminate with thickness large.chromium.png|0| Added|
|ProgressBar converged.Indeterminate.chromium.png|0| Added|
|ProgressBar converged.Success.chromium.png|0| Added|
|ProgressBar converged.Warning.chromium.png|0| Added|
|Radio Converged.checked - Dark Mode.active.chromium.png|0| Added|
|Radio Converged.checked - Dark Mode.hover.chromium.png|0| Added|
|Radio Converged.checked - Dark Mode.rest.chromium.png|0| Added|
|Radio Converged.checked - High Contrast.active.chromium.png|0| Added|
|Radio Converged.checked - High Contrast.hover.chromium.png|0| Added|
|Radio Converged.checked - High Contrast.rest.chromium.png|0| Added|
|Radio Converged.checked.active.chromium.png|0| Added|
|Radio Converged.checked.hover.chromium.png|0| Added|
|Radio Converged.checked.rest.chromium.png|0| Added|
|Radio Converged.disabled - Dark Mode.active.chromium.png|0| Added|
|Radio Converged.disabled - Dark Mode.hover.chromium.png|0| Added|
|Radio Converged.disabled - Dark Mode.rest.chromium.png|0| Added|
|Radio Converged.disabled - High Contrast.active.chromium.png|0| Added|
|Radio Converged.disabled - High Contrast.hover.chromium.png|0| Added|
|Radio Converged.disabled - High Contrast.rest.chromium.png|0| Added|
|Radio Converged.disabled+checked - Dark Mode.default.chromium.png|0| Added|
|Radio Converged.disabled+checked - High Contrast.default.chromium.png|0| Added|
|Radio Converged.disabled+checked.default.chromium.png|0| Added|
|Radio Converged.disabled.active.chromium.png|0| Added|
|Radio Converged.disabled.hover.chromium.png|0| Added|
|Radio Converged.disabled.rest.chromium.png|0| Added|
|Radio Converged.group-disabled - RTL.default.chromium.png|0| Added|
|Radio Converged.group-disabled.default.chromium.png|0| Added|
|Radio Converged.group-horizontal - RTL.default.chromium.png|0| Added|
|Radio Converged.group-horizontal-stacked - RTL.default.chromium.png|0| Added|
|Radio Converged.group-horizontal-stacked.default.chromium.png|0| Added|
|Radio Converged.group-horizontal.default.chromium.png|0| Added|
|Radio Converged.group-vertical - RTL.default.chromium.png|0| Added|
|Radio Converged.group-vertical.default.chromium.png|0| Added|
|Radio Converged.label-after - RTL.default.chromium.png|0| Added|
|Radio Converged.label-after.default.chromium.png|0| Added|
|Radio Converged.label-below - RTL.default.chromium.png|0| Added|
|Radio Converged.label-below.default.chromium.png|0| Added|
|Radio Converged.label-wrapping - RTL.default.chromium.png|0| Added|
|Radio Converged.label-wrapping.default.chromium.png|0| Added|
|Radio Converged.no-label - RTL.default.chromium.png|0| Added|
|Radio Converged.no-label.default.chromium.png|0| Added|
|Radio Converged.required - RTL.default.chromium.png|0| Added|
|Radio Converged.required.default.chromium.png|0| Added|
|Radio Converged.unchecked - Dark Mode.active.chromium.png|0| Added|
|Radio Converged.unchecked - Dark Mode.hover.chromium.png|0| Added|
|Radio Converged.unchecked - Dark Mode.rest.chromium.png|0| Added|
|Radio Converged.unchecked - High Contrast.active.chromium.png|0| Added|
|Radio Converged.unchecked - High Contrast.hover.chromium.png|0| Added|
|Radio Converged.unchecked - High Contrast.rest.chromium.png|0| Added|
|Radio Converged.unchecked.active.chromium.png|0| Added|
|Radio Converged.unchecked.hover.chromium.png|0| Added|
|Radio Converged.unchecked.rest.chromium.png|0| Added|
|Select Converged.Appearance- filled-darker.default.chromium.png|0| Added|
|Select Converged.Appearance- filled-darker.focused.chromium.png|0| Added|
|Select Converged.Appearance- filled-darker.hover.chromium.png|0| Added|
|Select Converged.Appearance- filled-lighter.default.chromium.png|0| Added|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|0| Added|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|0| Added|
|Select Converged.Appearance- outline (default).default.chromium.png|0| Added|
|Select Converged.Appearance- outline (default).focused.chromium.png|0| Added|
|Select Converged.Appearance- outline (default).hover.chromium.png|0| Added|
|Select Converged.Appearance- underline.default.chromium.png|0| Added|
|Select Converged.Appearance- underline.focused.chromium.png|0| Added|
|Select Converged.Appearance- underline.hover.chromium.png|0| Added|
|Select Converged.Custom Icon.default.chromium.png|0| Added|
|Select Converged.Disabled.default.chromium.png|0| Added|
|Select Converged.Invalid- filled-darker.default.chromium.png|0| Added|
|Select Converged.Invalid- filled-darker.focused.chromium.png|0| Added|
|Select Converged.Invalid- filled-darker.hover.chromium.png|0| Added|
|Select Converged.Invalid- filled-lighter.default.chromium.png|0| Added|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|0| Added|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|0| Added|
|Select Converged.Invalid- outline.default.chromium.png|0| Added|
|Select Converged.Invalid- outline.focused.chromium.png|0| Added|
|Select Converged.Invalid- outline.hover.chromium.png|0| Added|
|Select Converged.Invalid- underline.default.chromium.png|0| Added|
|Select Converged.Invalid- underline.focused.chromium.png|0| Added|
|Select Converged.Invalid- underline.hover.chromium.png|0| Added|
|Select Converged.Size- large.default.chromium.png|0| Added|
|Select Converged.Size- small.default.chromium.png|0| Added|
|Select Converged.With appearance override.default.chromium.png|0| Added|
|Select Converged.With value.default.chromium.png|0| Added|
|Slider Converged.Disabled - Dark Mode.default.chromium.png|0| Added|
|Slider Converged.Disabled - Dark Mode.hover.chromium.png|0| Added|
|Slider Converged.Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Slider Converged.Disabled - High Contrast.default.chromium.png|0| Added|
|Slider Converged.Disabled - High Contrast.hover.chromium.png|0| Added|
|Slider Converged.Disabled - High Contrast.pressed.chromium.png|0| Added|
|Slider Converged.Disabled Vertical.default.chromium.png|0| Added|
|Slider Converged.Disabled Vertical.hover.chromium.png|0| Added|
|Slider Converged.Disabled Vertical.pressed.chromium.png|0| Added|
|Slider Converged.Disabled.default.chromium.png|0| Added|
|Slider Converged.Disabled.hover.chromium.png|0| Added|
|Slider Converged.Disabled.pressed.chromium.png|0| Added|
|Slider Converged.Horizontal - 0% - RTL.default.chromium.png|0| Added|
|Slider Converged.Horizontal - 0%.default.chromium.png|0| Added|
|Slider Converged.Horizontal - 100% - RTL.default.chromium.png|0| Added|
|Slider Converged.Horizontal - 100%.default.chromium.png|0| Added|
|Slider Converged.Root - Dark Mode.default.chromium.png|0| Added|
|Slider Converged.Root - Dark Mode.hover.chromium.png|0| Added|
|Slider Converged.Root - Dark Mode.pressed.chromium.png|0| Added|
|Slider Converged.Root - High Contrast.default.chromium.png|0| Added|
|Slider Converged.Root - High Contrast.hover.chromium.png|0| Added|
|Slider Converged.Root - High Contrast.pressed.chromium.png|0| Added|
|Slider Converged.Root - RTL.default.chromium.png|0| Added|
|Slider Converged.Root - RTL.hover.chromium.png|0| Added|
|Slider Converged.Root - RTL.pressed.chromium.png|0| Added|
|Slider Converged.Root.default.chromium.png|0| Added|
|Slider Converged.Root.hover.chromium.png|0| Added|
|Slider Converged.Root.pressed.chromium.png|0| Added|
|Slider Converged.Vertical - 0% - RTL.default.chromium.png|0| Added|
|Slider Converged.Vertical - 0%.default.chromium.png|0| Added|
|Slider Converged.Vertical - 100% - RTL.default.chromium.png|0| Added|
|Slider Converged.Vertical - 100%.default.chromium.png|0| Added|
|Slider Converged.Vertical - RTL.default.chromium.png|0| Added|
|Slider Converged.Vertical - RTL.hover.chromium.png|0| Added|
|Slider Converged.Vertical - RTL.pressed.chromium.png|0| Added|
|Slider Converged.Vertical.default.chromium.png|0| Added|
|Slider Converged.Vertical.hover.chromium.png|0| Added|
|Slider Converged.Vertical.pressed.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.focused.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Appearance- underline.rest.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.focused.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Max Bound.rest.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.focused.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.At Min Bound.rest.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.Disabled.focused.chromium.png|0| Added|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Disabled.rest.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|0| Added|
|SpinButton Converged.Display Value.focused.chromium.png|0| Added|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Display Value.rest.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.focused.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- outline.rest.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.focused.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|0| Added|
|SpinButton Converged.Invalid- underline.rest.chromium.png|0| Added|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|0| Added|
|SpinButton Converged.Size- medium (default).default.chromium.png|0| Added|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|0| Added|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|0| Added|
|SpinButton Converged.Size- small - RTL.default.chromium.png|0| Added|
|SpinButton Converged.Size- small.default.chromium.png|0| Added|
|SpinButton Converged.With appearance override.default.chromium.png|0| Added|
|Spinner converged.Inverted - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Huge - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Huge - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Huge.chromium.png|0| Added|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Label Above - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Label Above - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Label Above.chromium.png|0| Added|
|Spinner converged.Inverted with Label After - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Label After - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Label After.chromium.png|0| Added|
|Spinner converged.Inverted with Label Before - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Label Before - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Label Before.chromium.png|0| Added|
|Spinner converged.Inverted with Label Below - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Label Below - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Label Below.chromium.png|0| Added|
|Spinner converged.Inverted with Label.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Large - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Large.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Small - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Small.chromium.png|0| Added|
|Spinner converged.Inverted with Size Large - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Size Large - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Size Large.chromium.png|0| Added|
|Spinner converged.Inverted with Size Medium - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Size Medium - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Size Medium.chromium.png|0| Added|
|Spinner converged.Inverted with Size Small - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted with Size Small - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted with Size Small.chromium.png|0| Added|
|Spinner converged.Inverted.chromium.png|0| Added|
|Spinner converged.Primary - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Huge - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Huge - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Huge.chromium.png|0| Added|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Label - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Label Above - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Label Above - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Label Above.chromium.png|0| Added|
|Spinner converged.Primary with Label After - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Label After - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Label After - RTL.chromium.png|0| Added|
|Spinner converged.Primary with Label After.chromium.png|0| Added|
|Spinner converged.Primary with Label Before - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Label Before - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Label Before - RTL.chromium.png|0| Added|
|Spinner converged.Primary with Label Before.chromium.png|0| Added|
|Spinner converged.Primary with Label Below - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Label Below - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Label Below.chromium.png|0| Added|
|Spinner converged.Primary with Label.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Large - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Large - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Large.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Small - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Small - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Small.chromium.png|0| Added|
|Spinner converged.Primary with Size Large - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Size Large - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Size Large.chromium.png|0| Added|
|Spinner converged.Primary with Size Medium - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Size Medium - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Size Medium.chromium.png|0| Added|
|Spinner converged.Primary with Size Small - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Size Small - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Size Small.chromium.png|0| Added|
|Spinner converged.Primary with Size Tiny - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary with Size Tiny - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary with Size Tiny.chromium.png|0| Added|
|Spinner converged.Primary.chromium.png|0| Added|
|Switch Converged.Disabled and checked - Dark Mode.default.chromium.png|0| Added|
|Switch Converged.Disabled and checked - Dark Mode.hover.chromium.png|0| Added|
|Switch Converged.Disabled and checked - Dark Mode.pressed.chromium.png|0| Added|
|Switch Converged.Disabled and checked - High Contrast.default.chromium.png|0| Added|
|Switch Converged.Disabled and checked - High Contrast.hover.chromium.png|0| Added|
|Switch Converged.Disabled and checked - High Contrast.pressed.chromium.png|0| Added|
|Switch Converged.Disabled and checked.default.chromium.png|0| Added|
|Switch Converged.Disabled and checked.hover.chromium.png|0| Added|
|Switch Converged.Disabled and checked.pressed.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked - Dark Mode.default.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked - Dark Mode.hover.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked - Dark Mode.pressed.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked - High Contrast.default.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked - High Contrast.hover.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked - High Contrast.pressed.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked.default.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked.hover.chromium.png|0| Added|
|Switch Converged.Disabled and unchecked.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and checked - Dark Mode.default.chromium.png|0| Added|
|Switch Converged.Enabled and checked - Dark Mode.hover.chromium.png|0| Added|
|Switch Converged.Enabled and checked - Dark Mode.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and checked - High Contrast.default.chromium.png|0| Added|
|Switch Converged.Enabled and checked - High Contrast.hover.chromium.png|0| Added|
|Switch Converged.Enabled and checked - High Contrast.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and checked - RTL.default.chromium.png|0| Added|
|Switch Converged.Enabled and checked - RTL.hover.chromium.png|0| Added|
|Switch Converged.Enabled and checked - RTL.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and checked.default.chromium.png|0| Added|
|Switch Converged.Enabled and checked.hover.chromium.png|0| Added|
|Switch Converged.Enabled and checked.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - Dark Mode.default.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - Dark Mode.hover.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - Dark Mode.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - High Contrast.default.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - High Contrast.hover.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - High Contrast.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - RTL.default.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - RTL.hover.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked - RTL.pressed.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked.default.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked.hover.chromium.png|0| Added|
|Switch Converged.Enabled and unchecked.pressed.chromium.png|0| Added|
|Switch Converged.Required.default.chromium.png|0| Added|
|Switch Converged.Required.hover.chromium.png|0| Added|
|Switch Converged.Required.pressed.chromium.png|0| Added|
|Switch Converged.With label above - RTL.default.chromium.png|0| Added|
|Switch Converged.With label above - RTL.hover.chromium.png|0| Added|
|Switch Converged.With label above - RTL.pressed.chromium.png|0| Added|
|Switch Converged.With label above.default.chromium.png|0| Added|
|Switch Converged.With label above.hover.chromium.png|0| Added|
|Switch Converged.With label above.pressed.chromium.png|0| Added|
|Switch Converged.With label after - RTL.default.chromium.png|0| Added|
|Switch Converged.With label after - RTL.hover.chromium.png|0| Added|
|Switch Converged.With label after - RTL.pressed.chromium.png|0| Added|
|Switch Converged.With label after.default.chromium.png|0| Added|
|Switch Converged.With label after.hover.chromium.png|0| Added|
|Switch Converged.With label after.pressed.chromium.png|0| Added|
|Switch Converged.With label before - RTL.default.chromium.png|0| Added|
|Switch Converged.With label before - RTL.hover.chromium.png|0| Added|
|Switch Converged.With label before - RTL.pressed.chromium.png|0| Added|
|Switch Converged.With label before.default.chromium.png|0| Added|
|Switch Converged.With label before.hover.chromium.png|0| Added|
|Switch Converged.With label before.pressed.chromium.png|0| Added|
|Switch Converged.With label wrapping.default.chromium.png|0| Added|
|Switch Converged.With label wrapping.hover.chromium.png|0| Added|
|Switch Converged.With label wrapping.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Default - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Default - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Default - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Default - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Default - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Default - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Default - RTL.default.chromium.png|0| Added|
|TabList and Tab Converged.Default - RTL.hover.chromium.png|0| Added|
|TabList and Tab Converged.Default - RTL.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Default.default.chromium.png|0| Added|
|TabList and Tab Converged.Default.hover.chromium.png|0| Added|
|TabList and Tab Converged.Default.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Large size - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Large size - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Large size - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Large size - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Large size - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Large size - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Large size.default.chromium.png|0| Added|
|TabList and Tab Converged.Large size.hover.chromium.png|0| Added|
|TabList and Tab Converged.Large size.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Small size - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Small size - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Small size - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Small size - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Small size - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Small size - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Small size.default.chromium.png|0| Added|
|TabList and Tab Converged.Small size.hover.chromium.png|0| Added|
|TabList and Tab Converged.Small size.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance.default.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance.hover.chromium.png|0| Added|
|TabList and Tab Converged.Subtle appearance.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default) - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default) - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default).default.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default).hover.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected (default).pressed.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected.default.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected.hover.chromium.png|0| Added|
|TabList and Tab Converged.Tab Selected.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and large size.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size - Dark Mode.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size - Dark Mode.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size - Dark Mode.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size - High Contrast.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size - High Contrast.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size - High Contrast.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical and small size.pressed.chromium.png|0| Added|
|TabList and Tab Converged.Vertical.default.chromium.png|0| Added|
|TabList and Tab Converged.Vertical.hover.chromium.png|0| Added|
|TabList and Tab Converged.Vertical.pressed.chromium.png|0| Added|
|TabList and Tab Converged.With icon - RTL.default.chromium.png|0| Added|
|TabList and Tab Converged.With icon - RTL.hover.chromium.png|0| Added|
|TabList and Tab Converged.With icon - RTL.pressed.chromium.png|0| Added|
|TabList and Tab Converged.With icon and vertical - RTL.default.chromium.png|0| Added|
|TabList and Tab Converged.With icon and vertical - RTL.hover.chromium.png|0| Added|
|TabList and Tab Converged.With icon and vertical - RTL.pressed.chromium.png|0| Added|
|TabList and Tab Converged.With icon and vertical.default.chromium.png|0| Added|
|TabList and Tab Converged.With icon and vertical.hover.chromium.png|0| Added|
|TabList and Tab Converged.With icon and vertical.pressed.chromium.png|0| Added|
|TabList and Tab Converged.With icon only and vertical.default.chromium.png|0| Added|
|TabList and Tab Converged.With icon only and vertical.hover.chromium.png|0| Added|
|TabList and Tab Converged.With icon only and vertical.pressed.chromium.png|0| Added|
|TabList and Tab Converged.With icon only.default.chromium.png|0| Added|
|TabList and Tab Converged.With icon only.hover.chromium.png|0| Added|
|TabList and Tab Converged.With icon only.pressed.chromium.png|0| Added|
|TabList and Tab Converged.With icon.default.chromium.png|0| Added|
|TabList and Tab Converged.With icon.hover.chromium.png|0| Added|
|TabList and Tab Converged.With icon.pressed.chromium.png|0| Added|
|Table flex - subtle selection.rest.chromium.png|0| Added|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|0| Added|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|0| Added|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|0| Added|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.always visible - RTL.chromium.png|0| Added|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.always visible.chromium.png|0| Added|
|Table layout flex - cell actions.always visible.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|0| Added|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.default - High Contrast.chromium.png|0| Added|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.default - RTL.chromium.png|0| Added|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.default.chromium.png|0| Added|
|Table layout flex - cell actions.default.hover row.chromium.png|0| Added|
|Table layout flex - cell actions.in header cell.chromium.png|0| Added|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|0| Added|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|0| Added|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|0| Added|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|0| Added|
|Table layout flex - headers.sortable - High Contrast.chromium.png|0| Added|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|0| Added|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|0| Added|
|Table layout flex - headers.sortable.chromium.png|0| Added|
|Table layout flex - headers.sortable.hover header.chromium.png|0| Added|
|Table layout flex - headers.sortable.press header.chromium.png|0| Added|
|Table layout flex - truncate.default (disabled).chromium.png|0| Added|
|Table layout flex - truncate.false.chromium.png|0| Added|
|Table layout flex - truncate.true.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - brand.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|0| Added|
|Table layout flex.multiselect (checked) - neutral.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - brand.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|0| Added|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|0| Added|
|Table layout flex.multiselect - Dark Mode.chromium.png|0| Added|
|Table layout flex.multiselect - High Contrast.chromium.png|0| Added|
|Table layout flex.multiselect - RTL.chromium.png|0| Added|
|Table layout flex.multiselect.chromium.png|0| Added|
|Table layout flex.primary cell.chromium.png|0| Added|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|0| Added|
|Table layout flex.single select (checked) - High Contrast.chromium.png|0| Added|
|Table layout flex.single select (checked) - RTL.chromium.png|0| Added|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|0| Added|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|0| Added|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|0| Added|
|Table layout flex.single select (checked) - brand.chromium.png|0| Added|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|0| Added|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|0| Added|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|0| Added|
|Table layout flex.single select (checked) - neutral.chromium.png|0| Added|
|Table layout flex.single select (checked).chromium.png|0| Added|
|Table layout flex.single select - Dark Mode.chromium.png|0| Added|
|Table layout flex.single select - High Contrast.chromium.png|0| Added|
|Table layout flex.single select - RTL.chromium.png|0| Added|
|Table layout flex.single select.chromium.png|0| Added|
|Table layout flex.size - extra small.chromium.png|0| Added|
|Table layout flex.size - medium - Dark Mode.chromium.png|0| Added|
|Table layout flex.size - medium - High Contrast.chromium.png|0| Added|
|Table layout flex.size - medium - RTL.chromium.png|0| Added|
|Table layout flex.size - medium.chromium.png|0| Added|
|Table layout flex.size - small.chromium.png|0| Added|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|0| Added|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|0| Added|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|0| Added|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|0| Added|
|Table layout table - cell actions.always visible - RTL.chromium.png|0| Added|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|0| Added|
|Table layout table - cell actions.always visible.chromium.png|0| Added|
|Table layout table - cell actions.always visible.hover row.chromium.png|0| Added|
|Table layout table - cell actions.default - Dark Mode.chromium.png|0| Added|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|0| Added|
|Table layout table - cell actions.default - High Contrast.chromium.png|0| Added|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|0| Added|
|Table layout table - cell actions.default - RTL.chromium.png|0| Added|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|0| Added|
|Table layout table - cell actions.default.chromium.png|0| Added|
|Table layout table - cell actions.default.hover row.chromium.png|0| Added|
|Table layout table - cell actions.in header cell.chromium.png|0| Added|
|Table layout table - cell actions.in header cell.hover row.chromium.png|0| Added|
|Table layout table - headers.sortable - Dark Mode.chromium.png|0| Added|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|0| Added|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|0| Added|
|Table layout table - headers.sortable - High Contrast.chromium.png|0| Added|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|0| Added|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|0| Added|
|Table layout table - headers.sortable.chromium.png|0| Added|
|Table layout table - headers.sortable.hover header.chromium.png|0| Added|
|Table layout table - headers.sortable.press header.chromium.png|0| Added|
|Table layout table - truncate.default (disabled).chromium.png|0| Added|
|Table layout table - truncate.false.chromium.png|0| Added|
|Table layout table - truncate.true.chromium.png|0| Added|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|0| Added|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|0| Added|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|0| Added|
|Table layout table.multiselect (checked) - brand.chromium.png|0| Added|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|0| Added|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|0| Added|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|0| Added|
|Table layout table.multiselect (checked) - neutral.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - brand.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|0| Added|
|Table layout table.multiselect (mixed) - neutral.chromium.png|0| Added|
|Table layout table.multiselect - Dark Mode.chromium.png|0| Added|
|Table layout table.multiselect - High Contrast.chromium.png|0| Added|
|Table layout table.multiselect - RTL.chromium.png|0| Added|
|Table layout table.multiselect.chromium.png|0| Added|
|Table layout table.primary cell.chromium.png|0| Added|
|Table layout table.single select (checked) - Dark Mode.chromium.png|0| Added|
|Table layout table.single select (checked) - High Contrast.chromium.png|0| Added|
|Table layout table.single select (checked) - RTL.chromium.png|0| Added|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|0| Added|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|0| Added|
|Table layout table.single select (checked) - brand - RTL.chromium.png|0| Added|
|Table layout table.single select (checked) - brand.chromium.png|0| Added|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|0| Added|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|0| Added|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|0| Added|
|Table layout table.single select (checked) - neutral.chromium.png|0| Added|
|Table layout table.single select (checked).chromium.png|0| Added|
|Table layout table.single select - Dark Mode.chromium.png|0| Added|
|Table layout table.single select - High Contrast.chromium.png|0| Added|
|Table layout table.single select - RTL.chromium.png|0| Added|
|Table layout table.single select.chromium.png|0| Added|
|Table layout table.size - extra small.chromium.png|0| Added|
|Table layout table.size - medium - Dark Mode.chromium.png|0| Added|
|Table layout table.size - medium - High Contrast.chromium.png|0| Added|
|Table layout table.size - medium - RTL.chromium.png|0| Added|
|Table layout table.size - medium.chromium.png|0| Added|
|Table layout table.size - small.chromium.png|0| Added|
|Table table - subtle selection.rest.chromium.png|0| Added|
|Table table - subtle selection.rest.hover unselected row.chromium.png|0| Added|
|Text Converged.Default - Dark Mode.normal.chromium.png|0| Added|
|Text Converged.Default - High Contrast.normal.chromium.png|0| Added|
|Text Converged.Default - RTL.normal.chromium.png|0| Added|
|Text Converged.Default.normal.chromium.png|0| Added|
|Text Converged.Typography presets - RTL.normal.chromium.png|0| Added|
|Text Converged.Typography presets.normal.chromium.png|0| Added|
|Textarea Converged.Appearance- filled-darker.default.chromium.png|0| Added|
|Textarea Converged.Appearance- filled-darker.focused.chromium.png|0| Added|
|Textarea Converged.Appearance- filled-darker.hover.chromium.png|0| Added|
|Textarea Converged.Appearance- filled-lighter.default.chromium.png|0| Added|
|Textarea Converged.Appearance- filled-lighter.focused.chromium.png|0| Added|
|Textarea Converged.Appearance- filled-lighter.hover.chromium.png|0| Added|
|Textarea Converged.Appearance- outline (default).default.chromium.png|0| Added|
|Textarea Converged.Appearance- outline (default).focused.chromium.png|0| Added|
|Textarea Converged.Appearance- outline (default).hover.chromium.png|0| Added|
|Textarea Converged.Disabled.default.chromium.png|0| Added|
|Textarea Converged.Disabled.focused.chromium.png|0| Added|
|Textarea Converged.Disabled.hover.chromium.png|0| Added|
|Textarea Converged.Invalid- filled-darker.default.chromium.png|0| Added|
|Textarea Converged.Invalid- filled-darker.focused.chromium.png|0| Added|
|Textarea Converged.Invalid- filled-darker.hover.chromium.png|0| Added|
|Textarea Converged.Invalid- filled-lighter.default.chromium.png|0| Added|
|Textarea Converged.Invalid- filled-lighter.focused.chromium.png|0| Added|
|Textarea Converged.Invalid- filled-lighter.hover.chromium.png|0| Added|
|Textarea Converged.Invalid- outline.default.chromium.png|0| Added|
|Textarea Converged.Invalid- outline.focused.chromium.png|0| Added|
|Textarea Converged.Invalid- outline.hover.chromium.png|0| Added|
|Textarea Converged.Size- large.default.chromium.png|0| Added|
|Textarea Converged.Size- medium.default.chromium.png|0| Added|
|Textarea Converged.Size- small.default.chromium.png|0| Added|
|Textarea Converged.With appearance override.default.chromium.png|0| Added|
|Textarea Converged.With value.default.chromium.png|0| Added|
|Textarea Converged.With value.focused.chromium.png|0| Added|
|Textarea Converged.With value.hover.chromium.png|0| Added|
|ToggleButton Converged.Checked.default.chromium.png|0| Added|
|ToggleButton Converged.Checked.hover.chromium.png|0| Added|
|ToggleButton Converged.Checked.pressed.chromium.png|0| Added|
|ToggleButton Converged.Circular and icon only.default.chromium.png|0| Added|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|0| Added|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|0| Added|
|ToggleButton Converged.Circular.default.chromium.png|0| Added|
|ToggleButton Converged.Circular.hover.chromium.png|0| Added|
|ToggleButton Converged.Circular.pressed.chromium.png|0| Added|
|ToggleButton Converged.Default - Dark Mode.default.chromium.png|0| Added|
|ToggleButton Converged.Default - Dark Mode.hover.chromium.png|0| Added|
|ToggleButton Converged.Default - Dark Mode.pressed.chromium.png|0| Added|
|ToggleButton Converged.Default - High Contrast.default.chromium.png|0| Added|
|ToggleButton Converged.Default - High Contrast.hover.chromium.png|0| Added|
|ToggleButton Converged.Default - High Contrast.pressed.chromium.png|0| Added|
|ToggleButton Converged.Default - RTL.default.chromium.png|0| Added|
|ToggleButton Converged.Default - RTL.hover.chromium.png|0| Added|
|ToggleButton Converged.Default - RTL.pressed.chromium.png|0| Added|
|ToggleButton Converged.Default.default.chromium.png|0| Added|
|ToggleButton Converged.Default.hover.chromium.png|0| Added|
|ToggleButton Converged.Default.pressed.chromium.png|0| Added|
|ToggleButton Converged.Disabled.default.chromium.png|0| Added|
|ToggleButton Converged.Disabled.hover.chromium.png|0| Added|
|ToggleButton Converged.Disabled.pressed.chromium.png|0| Added|
|ToggleButton Converged.Icon only.default.chromium.png|0| Added|
|ToggleButton Converged.Icon only.hover.chromium.png|0| Added|
|ToggleButton Converged.Icon only.pressed.chromium.png|0| Added|
|ToggleButton Converged.Outline.default.chromium.png|0| Added|
|ToggleButton Converged.Outline.hover.chromium.png|0| Added|
|ToggleButton Converged.Outline.pressed.chromium.png|0| Added|
|ToggleButton Converged.Primary - Dark Mode.default.chromium.png|0| Added|
|ToggleButton Converged.Primary - Dark Mode.hover.chromium.png|0| Added|
|ToggleButton Converged.Primary - Dark Mode.pressed.chromium.png|0| Added|
|ToggleButton Converged.Primary - High Contrast.default.chromium.png|0| Added|
|ToggleButton Converged.Primary - High Contrast.hover.chromium.png|0| Added|
|ToggleButton Converged.Primary - High Contrast.pressed.chromium.png|0| Added|
|ToggleButton Converged.Primary Checked.default.chromium.png|0| Added|
|ToggleButton Converged.Primary Checked.hover.chromium.png|0| Added|
|ToggleButton Converged.Primary Checked.pressed.chromium.png|0| Added|
|ToggleButton Converged.Primary Disabled.default.chromium.png|0| Added|
|ToggleButton Converged.Primary Disabled.hover.chromium.png|0| Added|
|ToggleButton Converged.Primary Disabled.pressed.chromium.png|0| Added|
|ToggleButton Converged.Primary.default.chromium.png|0| Added|
|ToggleButton Converged.Primary.hover.chromium.png|0| Added|
|ToggleButton Converged.Primary.pressed.chromium.png|0| Added|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|0| Added|
|ToggleButton Converged.Size large.default.chromium.png|0| Added|
|ToggleButton Converged.Size large.hover.chromium.png|0| Added|
|ToggleButton Converged.Size large.pressed.chromium.png|0| Added|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|0| Added|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|0| Added|
|ToggleButton Converged.Size small.default.chromium.png|0| Added|
|ToggleButton Converged.Size small.hover.chromium.png|0| Added|
|ToggleButton Converged.Size small.pressed.chromium.png|0| Added|
|ToggleButton Converged.Subtle Checked.default.chromium.png|0| Added|
|ToggleButton Converged.Subtle Checked.hover.chromium.png|0| Added|
|ToggleButton Converged.Subtle Checked.pressed.chromium.png|0| Added|
|ToggleButton Converged.Subtle Disabled.default.chromium.png|0| Added|
|ToggleButton Converged.Subtle Disabled.hover.chromium.png|0| Added|
|ToggleButton Converged.Subtle Disabled.pressed.chromium.png|0| Added|
|ToggleButton Converged.Subtle.default.chromium.png|0| Added|
|ToggleButton Converged.Subtle.hover.chromium.png|0| Added|
|ToggleButton Converged.Subtle.pressed.chromium.png|0| Added|
|ToggleButton Converged.Transparent Checked.default.chromium.png|0| Added|
|ToggleButton Converged.Transparent Checked.hover.chromium.png|0| Added|
|ToggleButton Converged.Transparent Checked.pressed.chromium.png|0| Added|
|ToggleButton Converged.Transparent Disabled.default.chromium.png|0| Added|
|ToggleButton Converged.Transparent Disabled.hover.chromium.png|0| Added|
|ToggleButton Converged.Transparent Disabled.pressed.chromium.png|0| Added|
|ToggleButton Converged.Transparent.default.chromium.png|0| Added|
|ToggleButton Converged.Transparent.hover.chromium.png|0| Added|
|ToggleButton Converged.Transparent.pressed.chromium.png|0| Added|
|ToggleButton Converged.With icon after content.default.chromium.png|0| Added|
|ToggleButton Converged.With icon after content.hover.chromium.png|0| Added|
|ToggleButton Converged.With icon after content.pressed.chromium.png|0| Added|
|ToggleButton Converged.With icon before content.default.chromium.png|0| Added|
|ToggleButton Converged.With icon before content.hover.chromium.png|0| Added|
|ToggleButton Converged.With icon before content.pressed.chromium.png|0| Added|
|Toolbar Converged.Default.Button Pressed.chromium.png|0| Added|
|Toolbar Converged.Default.Toggle On.chromium.png|0| Added|
|Toolbar Converged.Default.default.chromium.png|0| Added|
|Toolbar Converged.Far Group.chromium.png|0| Added|
|Toolbar Converged.Large.Button Pressed.chromium.png|0| Added|
|Toolbar Converged.Large.Toggle On.chromium.png|0| Added|
|Toolbar Converged.Large.default.chromium.png|0| Added|
|Toolbar Converged.Small.Button Pressed.chromium.png|0| Added|
|Toolbar Converged.Small.Toggle On.chromium.png|0| Added|
|Toolbar Converged.Small.default.chromium.png|0| Added|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|0| Added|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|0| Added|
|Toolbar Converged.Vertical Button.default.chromium.png|0| Added|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|0| Added|
|Toolbar Converged.Vertical.Toggle On.chromium.png|0| Added|
|Toolbar Converged.Vertical.default.chromium.png|0| Added|
|Tooltip Converged.basic - Dark Mode.chromium.png|0| Added|
|Tooltip Converged.basic - High Contrast.chromium.png|0| Added|
|Tooltip Converged.basic.chromium.png|0| Added|
|Tooltip Converged.inverted - Dark Mode.chromium.png|0| Added|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|0| Added|
|Tooltip Converged.inverted withArrow.chromium.png|0| Added|
|Tooltip Converged.inverted.chromium.png|0| Added|
|Tooltip Converged.positioning - High Contrast.chromium.png|0| Added|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|0| Added|
|Tooltip Converged.positioning - RTL.chromium.png|0| Added|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|0| Added|
|Tooltip Converged.positioning.chromium.png|0| Added|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|0| Added|
|Tooltip Converged.text-wrapping.chromium.png|0| Added|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|0| Added|
|Tooltip Converged.withArrow - High Contrast.chromium.png|0| Added|
|Tooltip Converged.withArrow.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>